BioinformaticTools-inator/BioinformaticToolsInator.py

```diff
@@ -0,0 +1,90 @@
+from typing import Dict, List, Tuple, Union
+from modules import dna_rna_funcs, fastq_funcs
+
+
+def run_dna_rna_tools(
+        *args: str
+        ) -> Union[List[str], Dict[str, Dict[str, int]], str]:
+    """
+    Performs a specified action on all sequences transmitted from input
+
+    Args:
+    - DNA or RNA sequences as str and a procedure to perform
+      as the last argument
+
+    Possible procedures:
+    - Main procedures: reverse, transcribe, complement, reverse_complement
+    - Additional procedures:
+      nucleotide_frequency, gc_content, find_start_codons
+
+    Returns:
+    - Union[List[str], Dict[str, Dict[str, int]], str]:
+      sequence or a collection of sequences with paramters
+    """
+    *seqs, operation = args
+    for seq in seqs:
+        dna_rna_funcs.is_valid_na(seq)
+    if operation == 'reverse':
+        return dna_rna_funcs.reverse(*seqs)
+    elif operation == 'transcribe':
+        return dna_rna_funcs.transcribe(*seqs)
+    elif operation == 'complement':
+        return dna_rna_funcs.complement(*seqs)
+    elif operation == 'reverse_complement':
+        return dna_rna_funcs.reverse_complement(*seqs)
+    elif operation == 'nucleotide_frequency':
+        return dna_rna_funcs.nucleotide_frequency(*seqs)
+    elif operation == 'gc_content':
+        return dna_rna_funcs.gc_content(*seqs)
+    elif operation == 'find_start_codons':
+        return dna_rna_funcs.find_start_codons(*seqs)
+    else:
+        print('Unidentified instruction')
```

Comment:
```suggestion
operations = {
    'reverse': dna_rna_funcs.reverse,
    'transcribe': dna_rna_funcs.transcribe,
    'complement': dna_rna_funcs.complement,
    'reverse_complement': dna_rna_funcs.reverse_complement,
    'nucleotide_frequency': dna_rna_funcs.nucleotide_frequency,
    'gc_content': dna_rna_funcs.gc_content,
    'find_start_codons': dna_rna_funcs.find_start_codons
}
if operation in operations:
    return operations[operation](*seqs)
else:
    raise ValueError(f"Unidentified instruction: {operation}")
```
Вместо длинной цепочки elifов можно все запихнуть в словарь
А в конце еще можно добавить обработку ошибок в случае некорректной последовательности или явный возврат ошибки с пояснением